src/components/reviews/reviewsList.jsx

```diff
@@ -7,30 +7,24 @@ class ReviewsList extends React.Component {
     super(props);
 
     this.state = {
-      reviewsShown: 0,
-      showMoreButton: true,
```

Comment:
Shifting to conditionally rendering seemed like the path I was going to end up going down with my buttons as well, reassuring to see that you're going down the same path